src/DataDog/AuditBundle/EventSubscriber/AuditSubscriber.php

```diff
@@ -70,9 +70,13 @@ public function getUnauditedEntities()
     {
         return array_keys($this->unauditedEntities);
     }
-
+    
     private function isEntityUnaudited($entity)
     {
+        if ($entity instanceof Association || $entity instanceof AuditLog) {
```

Comment:
no, because this entity is subclass of `Association` and `AuditLog`, although we could check if it is a `is_subclass_of($entitly, ...)`.